### ISSUE TITLE
For gene level models, do they work with rare variants only (i.e., is it required to set --freqUpper 0.05)? 

### ISSUE THREAD

**Issue by ppjeep** [zhanxw/rvtests#44]
In the example folder of rvtest package, you did not specify --freqUpper 0.05 in cmd.sh. Here is the example code. If I use --freqUpper 0.05 in the same command, the gene based results will be different. Is it OK if I run gene based analysis without using --freqUpper 0.05?
../executable/rvtest --pheno pheno --inVcf example.vcf.gz --setFile setFile --burden cmc,cmcWald,zeggini,zegginiWald --out out9

**Comment by zhanxw** [comment]
1. Usually rare variant test will use a upper frequency limit.
2. I don't have the reference. Actually, I learned the definition of LOF from my colleagues. Maybe @dajiangliu can give you a reference.

**Comment by dajiangliu** [comment]
I agree that synonymous variants may not be considered lof. For some of our own work, we group variants that are nonsynonymous, stop_gain or splice [see e.g. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5709146/] The choice of the annotation used to group variant is more of a personal taste, no right or wrong.
